source/common/event/BUILD

```diff
@@ -13,15 +16,24 @@ envoy_cc_library(
     srcs = [
         "dispatcher_impl.cc",
         "file_event_impl.cc",
-        "signal_impl.cc",
-    ],
-    hdrs = [
-        "signal_impl.h",
     ],
+    hdrs = select({
+        "//bazel:windows_x86_64": [
+            "win32/signal_impl.h",
+        ],
+        "//conditions:default": [
+            "posix/signal_impl.h",
+        ],
+    }),
```

Comment:
This header is provided by :signal_lib, could we remove it from the dispatcher library?

Reply:
I run into https://github.com/bazelbuild/bazel/issues/6337 if I remove it. This is why I added it

Reply:
Ah, ok.